vulpes/network/foxnet/effect.cpp

```diff
@@ -1,8 +1,19 @@
 /*
- * Vulpes (c) 2019 gbMichelle
+ * This file is part of Vulpes, an extension of Halo Custom Edition's capabilities.
+ * Copyright (C) 2019-2020 gbMichelle (Michelle van der Graaf)
  *
- * This program is free software under the GNU General Public License v3.0 or later. See LICENSE for more information.
+ * Vulpes is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, version 3.
+ *
+ * Vulpes is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * long with Vulpes.  If not, see <https://www.gnu.org/licenses/agpl-3.0.en.html>
  */
 
-spawn effect sig
-8B 4C 24 04 83 C8 FF 83 F9 FF 0F 84
+//spawn effect sig
+//8B 4C 24 04 83 C8 FF 83 F9 FF 0F 84
```

Comment:
danger

Reply:
I've debated just deleting this file tbh

Reply:
It's gone